### PR TITLE
WC: Rating component

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Added Rating component
-With hover change
-With a single "filled" rating star
-With "filled-siblings" rating star that also changes the color of the preceding rating stars

### CODE REVIEW

File: Documentation/Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    cursor: pointer;
+    font-size: 3rem;
+}
+
+.rating-star:hover,
+.rating-star:hover~.rating-star,
+.filled-siblings,
+.filled-siblings~*,
+.filled {
+    color: gold;
```
Comment (devananth) [inline Documentation/Components/Rating/rating.css]: `css variables` can be used.

**Review by devananth — COMMENTED** [review]
Everything else is fine.

File: Documentation/Components/Rating/rating.css (hunk 1 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/Rating/rating.css]: yeah, I forgot to change it. Thank you!

**Review by parthgabani777 — APPROVED** [review]
looks Good!

**Pull request merged**